src/types/onyx/PolicyCategory.ts

```diff
@@ -0,0 +1,18 @@
+type PolicyCategory = {
+    /** Name of a category */
+    name: string;
+
+    /** Flag that determines if a category is active and able to be selected */
+    enabled: boolean;
+
+    areCommentsRequired: boolean;
+
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    'GL Code': string;
+
+    externalID: string;
+
+    origin: string;
```

Comment:
@yuwenmemon, could you please clarify what those properties mean?
```suggestion
    areCommentsRequired: boolean;

    // eslint-disable-next-line @typescript-eslint/naming-convention
    'GL Code': string;

    externalID: string;

    origin: string;
```

Reply:
I asked @yuwenmemon above, there's no mention of them in the design doc, so I don't know how to describe them

Reply:
Yes, I understand. Let's just wait for him.

Reply:
Added in some docs as suggestions!